fastlane/Fastfile

```diff
@@ -512,34 +576,36 @@ UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(PROJECT_ROOT_FOLDER, '.configure-files
   end
 
   def screenshots_directory
-    File.join(Dir.pwd, "screenshots")
+    File.join(Dir.pwd, 'screenshots')
   end
 
   #####################################################################################
   # Private lanes
   #####################################################################################
-  main_strings_path = "./Simplenote/src/main/res/values/strings.xml"
-  update_strings_path = "./fastlane/resources/values/"
+  main_strings_path = './Simplenote/src/main/res/values/strings.xml'
+  update_strings_path = './fastlane/resources/values/'
```

Comment:
Use `File.join` instead

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/3267c3d07bb4efb667be614d849bb73e9b2b9ac3